### ISSUE TITLE
About padding index number

### ISSUE THREAD

**Issue by ys201810** [jaywonchung/BERT4Rec-VAE-Pytorch#17]
Hi!!

Thanks for great code!
I have one question about padding at dataloaders/bert.py.

https://github.com/jaywonchung/BERT4Rec-VAE-Pytorch/blob/f66f2534ebfd937778c7174b5f9f216efdebe5de/dataloaders/bert.py#L112
https://github.com/jaywonchung/BERT4Rec-VAE-Pytorch/blob/f66f2534ebfd937778c7174b5f9f216efdebe5de/dataloaders/bert.py#L146

At this code, if item length is not enough, then pad using [0].
But I think, index 0 already have a Corresponding itemID, so that is padding using real itemID, right?

Thanks

**Comment by ys201810** [comment]
Hi!

Thanks for your message.
Exactly your way is good, or use padding_id is item length + 1. ex)
padding_id = max(item_len) + 1
# If mask_id use max(item_len + 1) then padding_id = max(item_len) + 2
 something like that.

I really appreciated your code.
Thanks!!

2023年9月1日(金) 21:49 大锤哥 ***@***.***>:
